### PR TITLE
apps sc: added prometheus timeseries dashboard

### LINKED ISSUE #741: Add dashboard that shows how many metrics/time series we have
**Is your feature request related to a problem? Please describe.**
Having a lot of metrics and time series will cause our metrics components (prometheus, influxdb) to need more resources and eventually stop working if we do not increase the resources. It is currently also cumbersome to check how many timeseries there are in the cluster, you have to execute some manual queries in prometheus/influxdb.

**Describe the solution you'd like**
I would like a dashboard that shows the relevant numbers, preferably if they can also be filtered to show amount of time series per prometheus job or metric.

One important thing to note is that these queries can be quite resource intensive for prometheus to compute. So we need to make sure that is possible to use without a significant risk of killing prometheus (due to lack of resources) or a significant increase in allocated resources.

**Describe alternatives you've considered**
It could also be good to have an alert that warns if we have a high amount of metrics/time series. But then we also need to know what number to alert on, which might differ between clusters.

**Additional context**
Relevant prometheus queries to start with:
Count the number of time series per job: `sort_desc(count by(job) ({__name__!=""}))`
Count the number of time series per metric: `sort_desc(count by(__name__) ({__name__!=""}))`

**Definition of done:**
- We have a dashboard that can tell us relevant data around how many metrics/time series we have or some alternative solution that makes it easier to find and debug these problems.

### PR DESCRIPTION
**Pull request by davidumea**
**What this PR does / why we need it**:

Added a dashboard that shows how many time series we have in Prometheus.

**Which issue this PR fixes** : fixes #741 

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:
Did not make an alert for when there are too many time series which was something @viktor-f considered in #741.

**Checklist:**

- [X] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [X] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [X] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [X] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

### CODE REVIEW

**Comment by crssnd** [comment]
you should add the new dashboard [here](https://github.com/elastisys/compliantkubernetes-apps/blob/david/prometheus-dashboard/helmfile/charts/grafana-ops/values.yaml) and [here](https://github.com/elastisys/compliantkubernetes-apps/blob/david/prometheus-dashboard/helmfile/values/grafana-ops.yaml.gotmpl)
otherwise it will not be installed

File: helmfile/charts/grafana-ops/dashboards/prometheus-timeseries-dashboard.json [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+    ]
+  },
+  "time": {
+    "from": "now-15m",
+    "to": "now"
+  },
+  "timepicker": {
+    "hidden": true
+  },
+  "timezone": "",
```
Comment (crssnd) [inline helmfile/charts/grafana-ops/dashboards/prometheus-timeseries-dashboard.json:395]:
```suggestion
  "timezone": "utc",
```

File: helmfile/charts/grafana-ops/dashboards/prometheus-timeseries-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+  "schemaVersion": 32,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "prometheus-sc",
+          "value": "prometheus-sc"
```
Comment (crssnd) [inline helmfile/charts/grafana-ops/dashboards/prometheus-timeseries-dashboard.json:307]:
```suggestion
          "text": "default",
          "value": "default"
```

**Comment by davidumea** [comment]
> can you also delete the thresholds to total time series? or add some values to that, now everything is red

Should probably just delete them, since the threshold for "normal" values would be different for every cluster.

**Review by raviranjanelastisys — APPROVED** [review]
LGTM now !!

**Review by robinAwallace — APPROVED** [review]
LGTM

**Review by crssnd — APPROVED** [review]

**Pull request merged**